### DISCUSSION TITLE
Discuss: Pathologist Annotation Behavior Report

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#57] [Ideas]
Discuss, or Question and answer about this project specifically.

**Comment by BasmaElhoseny01** [comment]
Hello @birm  I am interested in that project i want to know how to start to see the alrwady running SW to be able to understand more about the requirments . How to reach the mentors to discuss.

**Reply by birm** [reply]
For running the exiting software, http://camicroscope.org/docs/ is _hopefully_ a good start. If not, please report anything confusing/missing/unclear in the docs and we'll hopefully update!

**Comment by kailash360** [comment]
Hello @birm 

I went through the problem statement and found it really interesting. We can define different types of behavior in the system, and collect logs of each of them. This behavioral data can be then stored in a database for further processing. I would love to work on this and discuss further to formulate the solution.

**Comment by YukaUU** [comment]
Hello, @birm
I am a second-year junior software developer working for a Japanese IT company! 
In the past two years, I have worked on three projects.
1. Built the front-end, back-end, and infrastructure of a Saas authentication management system for about 20,000 employees. 
2. Implemented a real-time communication feature to an online programming education service that learners can discuss together.
3. Established accessibility standards for web applications with a more user-friendly UI design for visually impaired employees.

I am highly motivated and looking forward to contributing to a broader dev project, like caMicroscope.I like this product very much and am interested in the tech stack behind the issues. 

My skills: JavaScript, TypeScript, HTML, CSS, React, Next.js, MUI, PostgresSQL, Prisma, Django, Flask, Python, Google Analytics, Ansible, Jenkins, Linux, Git

**Comment by Zarlanx** [comment]
Hello @birm 

My name is Khaled a Computer Science student at McGill and I am very interested in the "Pathologist Annotation Behaviour Report" project for caMicroscope. I have also worked for the past 3 years in the histology sector and I have used various slide scanning/image recognition software and getting feedback from pathologists.. I have been studying and understanding the code base for the past 2 days to gain a deeper understanding of its structure and how everything works together. I know I am bit late, but as I am looking at the project description I would like some further clarification as to what types of visualizations we are aiming for in the behaviour report. Are there any specific metrics or trends that we want to highlight? Understanding these details would greatly assist me in aligning my efforts with the project's goals. Thanks!
